_data/homepage.yml

```
@@ -22,17 +22,17 @@ hero-banner: /images/CSFBanner.jpg
 # key-highlights:
 # Key Highlights
 key-highlights:
-  - title: Foresight Conference 
-    description: 2019 Report
-    url: https://www.csf.gov.sg/foresight-conference/2019/
+  - title: Beyong the Pandemic: Global Foresight Perspectives 
```

@YuliaHart this line has a colon in it. you have to change it as such:
```suggestion
  - title: "Beyong the Pandemic: Global Foresight Perspectives"
```

---

thanks, edited!